haystack/components/embedders/sentence_transformers_document_embedder.py

```diff
@@ -14,9 +14,14 @@
 @component
 class SentenceTransformersDocumentEmbedder:
     """
-    A component for computing Document embeddings using Sentence Transformers models.
+    Calculates document embeddings using Sentence Transformers models.
 
-    Usage example:
+    It stores the embeddings in the `embedding` metadata field of each document.
+    You can also embed documents' metadata.
+    Use this component in indexing pipelines to embed documents from PreProcessors
```

Comment:
They don't have to be after PreProcessors necessarily, can be just straight-up input documents – do you think it makes sense to omit this? Doesn't have to be "input" ⬇️ 

```suggestion
    Use this component in indexing pipelines to embed input documents
```